.gitignore

```diff
@@ -1,3 +1,2 @@
 *.sw*
 crc-driver-libvirt-*
-vendor/
```

Comment:
Can we also have a  make target to sync the vendor and have a check similar to what we have in crc to make sure it never become out of sync?

Reply:
I added `make vendorcheck`, I'm unconvinced about adding `make vendor`, as it's just calling `go mod vendor`

Reply:
> > Can we have this [code-ready/machine#54](https://github.com/code-ready/machine/pull/54) merged before this one?
> 
> Ah sure, it's merged now, so I'll update the branch in a bit.

Should be ok now.